Ubuntu 22.04: build fail

Building on Ubuntu 22.04 results in following error

```
FAILED: bin/rofi-sim-listener 
< output ommited >
/usr/bin/ld: gazebosimTools/CMakeFiles/rofi-sim-listener.dir/listener.cpp.o: undefined reference to symbol '_ZN3tbb6detail2r114notify_waitersEm'
/usr/bin/ld: /lib/x86_64-linux-gnu/libtbb.so.12: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

--- a/suites/desktop/CMakeLists.txt
+++ b/suites/desktop/CMakeLists.txt
@@ -127,6 +127,9 @@ if (${BUILD_GAZEBO})
   # Workaround for bug in gazebo that causes problems on `g++11` with `-std=c++20`
   set(GAZEBO_INCLUDE_DIRS "$ENV{ROFI_ROOT}/gazebo_include" ${GAZEBO_INCLUDE_DIRS})
 
+  find_library(LibTBB NAMES tbb)
+  link_libraries(gazebo ${LibTBB})
+
   find_package(Protobuf REQUIRED)
   # Include Gazebo messages
   set(PROTOBUF_IMPORT_DIRS)

